app-crypt/loop-aes-losetup/loop-aes-losetup-2.34.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools toolchain-funcs flag-o-matic
+
+MY_PV="${PV/_/-}"
+MY_P="util-linux-${MY_PV}"
+LOOPAES_P="loop-AES-v3.7p"
+S="${WORKDIR}/${MY_P}"
+
+DESCRIPTION="Loop-AES losetup utility"
+HOMEPAGE="https://www.kernel.org/pub/linux/utils/util-linux/"
+SRC_URI="https://www.kernel.org/pub/linux/utils/util-linux/v${PV:0:4}/${MY_P}.tar.xz
```

Comment:
No way to use system util-linux?

Reply:
> No way to use system util-linux?

I don't think so.  The point of the loop-aes-losetup package is that the loop-aes kernel driver requires a modified losetup in order to use (to take extra parameters from the user, and pass them to the loop driver).  Upstream distributes this as a patch against util-linux, out of which you build only losetup (but rename it to avoid collision with system util-linux).

So loop-aes-losetup needs to be built from source, and, needs to track what version(s) of util-linux for which the loop-aes maintainer releases patches.